services/parser-database/parser.py

```diff
@@ -0,0 +1,230 @@
+import slate  # pylint: disable=import-error
+
+
+import retriever  # pylint: disable=import-error
+# import database  # pylint: disable=import-error
+import keywordMock  # pylint: disable=import-error
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Document:
+    """Class for Regulations Documents"""
+
+    def __init__(self, jurisdiction, shahash, url, last_updated):
+        self.id = None
+        self.jurisdiction = jurisdiction
+        self.shahash = shahash
+        self.url = url
+        self.last_updated = last_updated
+
+    @staticmethod
+    def from_dict():
+        pass
+
+    def to_dict(self):
+        pass
+
+    def __repr__(self):
+        return f"Document(\
+                id={self.id}, \
+                jurisdiction={self.jurisdiction}, \
+                last_updated={self.last_updated}, \
+                url={self.url}, \
+                shahash={self.shahash}\
+            )"
+
+
+class Article:
+    """Class for storing articles.
+    """
+
+    def __init__(self, number, text):
+        self.number = number
+        self.text = text
+
+
+articles_in_memory = []
+keywords_in_memory = {}
+
+
+def get_article_by_number(art_num):
+    for item in articles_in_memory:
+        if art_num == str(item["articleNumber"]):
+            return item
+    return "No article matches such ID", 402
+
+
+def get_articles_that_match_keywords(keywords_list):
+    """Returns articles that match the the given keyword(s)
+
+    Args:
+        keywords_list (list): Keyword(s) to look for
+
+    Returns:
+        list: articles that match such keyword(s)
+    """
+    to_return = []
+    for keyword in keywords_list:
+        if keyword in keywords_in_memory:
+            print(keyword)
+            to_return.append(keywords_in_memory[keyword])
+    return to_return
+
+
+def count_articles(pdf_text):
+    """Identifies articles and returns a list of Article objects.
+
+    Args:
+        pdf_text (string): contains the PDF text where articles
+        are to be identified.
+
+    Returns:
+        list: article objects
+    """
+    articles = []
+    article_text = ""
+    article_count = 1
+    i = 0
+    while i < len(pdf_text) - 1:
+        if (pdf_text[i] == "artículo" or pdf_text[i] == "articulo") and (
+                pdf_text[i + 1] == str(article_count) + ".-"
+                or pdf_text[i + 1] == str(article_count) + "-"
+                or pdf_text[i + 1] == str(article_count) + "."
+        ):
+            logging.info("Article #" + str(article_count) + " recognized!")
+            articles.append(Article(article_count, article_text))
+            article_text = ""
+            article_count += 1
+            i += 1
+        else:
+            article_text += " " + pdf_text[i]
+        i += 1
+    return articles
+
+
+mty_document = {
+    "hash": "afafbfbdce8c40924edae00f6ce54f0c639ce42a2" +
+    "c0fbbfa6ab82ea6925827c51",
+    "jurisdiction": "Monterrey",
+    "url": "http://www.guadalupe.gob.mx/wp-content/upl" +
```

Comment:
I'd leave it as a one-liner. https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
`pylint` gave me a very hard time with that, but didn't know about the error disable, will add `# pylint: disable=invalid-name`